tsweb/tsweb_test.go

```diff
@@ -514,6 +542,52 @@ foo_AUint16 65535
 	}
 }
 
+func TestVarzHandlerExponentialFloats(t *testing.T) {
+	t.Run("float_log", func(t *testing.T) {
+		rec := httptest.NewRecorder()
+		VarzHandler(rec, httptest.NewRequest("GET", "/", nil))
+		t.Logf("Got: %s", rec.Body.Bytes())
+	})
```

Comment:
this test doesn't test anything? leftover debug?

Reply:
Removed the whole function.